roles/addons/slurm/tasks/main.yml

```diff
@@ -19,46 +19,39 @@
         - "vars/{{ ansible_facts.os_family }}.yml"
       skip: true
 
-- name: Package
+- name: "package █ Install {{ slurm_packages_to_install[slurm.slurm_packaging][slurm_profile] }}"
   package:
     name: "{{ item }}"
```

Comment:
You can use the list here, no need to loop.

Reply:
missing tag package